admin.mss

```diff
@@ -1,34 +1,34 @@
-#disputed[zoom>=3][zoom<12] {
+#disputed{
   line-width: 1;
   line-color: @admin_3;
   line-clip: true;
   line-dasharray: 4, 4;
-  polygon-pattern-file: url('icon/pattern/disputed.svg');
-  polygon-pattern-alignment: local;
+  [zoom<12] {
+    polygon-pattern-file: url('icon/pattern/disputed.svg');
+    polygon-pattern-alignment: local;
+  }
   [zoom=9] {
     polygon-pattern-opacity: 0.8;
     line-opacity: 0.8;
   }
-  [zoom>=10] {
+  [zoom>=10][zoom<12] {
     polygon-pattern-opacity: 0.4;
     line-opacity: 0.4;
   }
 }
-#itl_boundary_low[zoom>=1][zoom<6][maritime!='yes'],
-#itl_boundary[zoom>=6] {
-  #itl_boundary[iso!='EH'], #itl_boundary_low[ISO3166-1!='EH']{
-    line-width: 0.5;
-    line-color: @admin_2;
-    line-clip: true;
-    [maritime='yes'] {
-      line-dasharray: 10,5;
-    }
-    [zoom>=4] {
-      line-width: 1;
-    }
-    [zoom>=10] {
-      line-width: 2;
-    }
+#itl_boundary_low[ISO3166-1!='EH'][zoom>=1][zoom<6][maritime!='yes'],
```

Comment:
Thinking about it a second time, why not filtering this out during the make_boundary step?

Reply:
Third though, this should not even be needed, given we don't have the Western Sahara in our CSV. Which at the end is all the list of country we process.

(And BTW, it's not a `boundary=administrative` but `boundary=disputed`: https://www.openstreetmap.org/relation/2559126

Reply:
Of course you are right, this was the problem

https://github.com/tilery/pianoforte/pull/20/commits/4f2dcde635ce882c8dd7fc38e49d96a4b17154b8#diff-1b8328945b14d8c22b127c55a04b3091L171